hikari/commands.py

```diff
@@ -181,6 +181,9 @@ class Command(snowflakes.Unique):
     This will be `builtins.None` if this is a global command.
     """
 
+    version: snowflakes.Snowflake = attr.field(eq=False, hash=False, repr=True)
+    """Auto-incrementing version identifier updated during seubstantial record changes."""
```

Comment:
This might confuse people reading it, might be worth changing it up a little and making it easier to understand

Reply:
what about it would confuse ppl

Reply:
I find personally confusing that the type of the attribute is a snowflake but the docstring suggests it should be an int, since it is incremented sequencially every deployment, nothing to do with a snowflake

Reply:
It is incremented from the commands Id, it's a snowflake